.github/workflows/lint-build-test.yml

```diff
@@ -104,6 +104,7 @@ jobs:
         uses: MetaMask/action-checkout-and-setup@v1
         with:
           is-high-risk-environment: false
+      - run: yarn build
```

Comment:
This is required to build the `@metamask/design-system-shared` package as it's a dependency of maskicon, jazzicon and blockies. I'm going to reach out to the @MetaMask/wallet-framework team to check this makes sense. Created an issue here https://consensyssoftware.atlassian.net/jira/software/c/projects/DSYS/boards/1888/backlog?assignee=6152e94cc7bea40069d6b9c3&selectedIssue=DSYS-166

```
ERROR: src/components/temp-components/Blockies/Blockies.tsx:1:39 - error TS2307: Cannot find module '@metamask/design-system-shared' or its corresponding type declarations.
```

<img width="1107" height="620" alt="Screenshot 2025-09-04 at 4 47 48 PM" src="https://github.com/user-attachments/assets/7f1ab1e8-46c6-49b9-b39c-784dd51a2583" />